ydb_sqlalchemy/dbapi/cursor.py

```diff
@@ -87,35 +80,40 @@ def __init__(
         driver: Union[ydb.Driver, ydb.aio.Driver],
         session_pool: Union[ydb.SessionPool, ydb.aio.SessionPool],
         tx_mode: ydb.AbstractTransactionModeBuilder,
+        request_timeout: datetime.timedelta,
```

Comment:
why don't we use whole BaseRequestSettings object here?

Reply:
Really... Fixed!